library/src/main/java/com/tolstykh/textviewrichdrawable/EditTextRichDrawable.java

```diff
@@ -45,4 +46,25 @@ public int getCompoundDrawableHeight() {
     public int getCompoundDrawableWidth() {
         return mRichDrawableHelper.getCompoundDrawableWidth();
     }
+
+    @Override
+    public void setDrawableStartVectorId(@DrawableRes int id) {
+        mRichDrawableHelper.setDrawableStartVectorId(id);
```

Comment:
You forgot to call `mRichDrawableHelper.apply(this);` inside each setter in this class